README.md

```diff
@@ -141,11 +152,25 @@ After training the parameters by either of the methods above, replace the parame
 
 ![image](https://github.com/open-spaced-repetition/fsrs4anki/assets/32575846/70b3b45a-f014-4574-81eb-cad6d19f93d9)
 
-⚠️Note: when replacing these parameters, make sure that you don't accidentally erase the square brackets or the comma after the closing bracket. Yes, the code will break without them.
+⚠️Note: when replacing these parameters, make sure that you don't accidentally erase the square brackets or the comma after the closing bracket. The code will break without them.
+
+#### Step 2.2: Choosing the desired retention rate and maximum interval
+
+Now, you need to choose your `requestRetention`, which denotes the retention rate (i.e. the fraction of the cards recalled successfully) that FSRS will try to achieve. 
+
+As an aid in deciding this value, you can view your past retention rate in Anki stats. For example, if your retention rate in the past was 90%, you can set 0.90 as your `requestRetention`.
+
+You can set a higher `requestRetention` but keep in mind that as you increase the `requestRetention` above 0.90, the review load (reviews/day) would increase very rapidly. For the same reason, it is not advisable to use a `requestRetention` greater than 0.97.
+
+After deciding the value of requestRetention, put this into the scheduler code. At the same time, decide the value of `maximumInterval`, which is the maximum interval any card is allowed to attain. The value in the FSRS scheduler code overrides the value set in Anki's deck options.
+
+![ Add screenshot ]
```

Comment:
@L-M-Sherlock, I have not added the screenshot here because I think that the screenshots should be similar to each other (i.e., all from a Mac). So, if you think that my version is good enough to be merged, please add a screenshot here.